sysutils/fluent-bit/Portfile

```diff
@@ -1,31 +1,34 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.1
+PortGroup           cmake   1.1
+PortGroup           github  1.0
 
-name                fluent-bit
-version             1.5.2
+github.setup        fluent fluent-bit 1.5.5 v
+revision            0
 categories          sysutils
 platforms           darwin
 license             Apache-2
-maintainers         {l2dy @l2dy} openmaintainer
+
+homepage            https://fluentbit.io/
+
 description         Fast and Lightweight Log processor and forwarder
+
 long_description    Fluent Bit is a Data Forwarder for Linux, Embedded Linux, \
                     OSX and BSD family operating systems. It's part of the \
                     Fluentd Ecosystem.  Fluent Bit allows collection of \
                     information from different sources, buffering and \
                     dispatching them to different outputs such as Fluentd, \
                     Elasticsearch, Nats or any HTTP end-point within others.
-homepage            https://fluentbit.io/
-set branch          [join [lrange [split ${version} .] 0 1] .]
-master_sites        ${homepage}releases/${branch}/
```

Comment:
Why not use the official tarball?

Reply:
I suppose you're right; changes appeared much sooner on Github as opposed to the official homepage.

Reply:
GitHub-generated git archives are not stable and their checksums may change. We prefer using official archives from the project maintainers, and in this case, from https://fluentbit.io/.

Reply:
Understood. Reverted to official archives.